src/io.c

```diff
@@ -92,6 +94,35 @@ io_set_process_status(mrb_state *mrb, pid_t pid, int status)
 }
 #endif
 
+// vargs version of mrb_sys_fail()
+static
+void mrb_io_fail(mrb_state *mrb, const char *format, ...)
```

Comment:
This is almost a copy of mrb_sys_fail()

cf. https://github.com/mruby/mruby/blob/master/src/error.c#L490